api/v1alpha1/httpchaos_types.go

```diff
@@ -134,12 +134,12 @@ func (in *HTTPChaos) IsDeleted() bool {
 	return !in.DeletionTimestamp.IsZero()
 }
 
-// IsPaused returns whether this resource has been paused
-func (in *HTTPChaos) IsPaused() bool {
-	if in.Annotations == nil || in.Annotations[PauseAnnotationKey] != "true" {
-		return false
+// GetPause returns whether this resource has been paused
+func (in *HTTPChaos) GetPause() string {
+	if in.Annotations == nil || in.Annotations[PauseAnnotationKey] == "" {
```

Comment:
We can simplify the logic here.

``` go
func (in *HTTPChaos) GetPause() string {
	if in.Annotations == nil  {
        }
        return in.Annotations[PauseAnnotationKey]
```

Reply:
Thank you for the suggestion. I thought it would be better to sacrifice code cleanness to improve the readability, like `if true` means it should not be paused, and vice versa. Don't know if that's needed.

Reply:
I didn't get your point. Why do u think this way doesn't have readability :(? In your context, the getPause method is to return the value of the pause annotation. So that should be super clear and readable

Reply:
I didn't get your point. Why do u think this way doesn't have readability :(? In your context, the getPause method is to return the value of the pause annotation. So that should be super clear and readable. Let's change.

Reply:
Thank you for the suggestion. That makes sense to me and I'll make change. 
I'm still new to this so I might have some bad coding habit :(